test/common/upstream/health_checker_impl_test.cc

```diff
@@ -95,6 +95,18 @@ TEST(HealthCheckerFactoryTest, CreateGrpc) {
                              .get()));
 }
 
+class HealthCheckerTestBase {
+public:
+  std::shared_ptr<MockClusterMockPrioritySet> cluster_;
+  NiceMock<Event::MockDispatcher> dispatcher_;
+  MockHealthCheckEventLogger* event_logger_{};
+  NiceMock<Runtime::MockRandomGenerator> random_;
+  NiceMock<Runtime::MockLoader> runtime_;
+  HealthCheckerTestBase()
+      : cluster_(new NiceMock<MockClusterMockPrioritySet>()),
```

Comment:
nit: std::make_shared<NiceMock<MockClusterMockPrioritySet>>(), and init-in-place rather than using a ctor.